advocacy_docs/edb-postgres-ai/cloud-service/index.mdx

```diff
@@ -16,6 +16,7 @@ redirects:
 - /purl/upm/home/
 - /biganimal/latest/
 - /biganimal/latest/overview/
+displayBanner: "Please be advised that the EDB Hosted Cloud Service has been deprecated. Support will still be available for current customers. However the related documentation topics removal work is in progress and will be removed shortly. We recommend affected users transition to alternative solutions as soon as possible. Further updates will be provided as the transition progresses."
```

Comment:
Not sure where this is supposed to appear, or it is only in the background. I cannot see any banners in the preview: 
https://deploy-preview-6315--edb-docs-staging.netlify.app/docs/edb-postgres-ai/cloud-service/ 

Also, I dropped an edited version of the message as suggestion in the comment below.

Reply:
Fixed issue with displayBanner.